<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1242</width>
    <height>810</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLineEdit" name="lineEdit"/>
              </item>
              <item>
               <widget class="QPushButton" name="loadButton">
                <property name="text">
                 <string>load</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QCheckBox" name="stepExe">
                  <property name="text">
                   <string>step exe</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="steps">
                  <property name="maximumSize">
                   <size>
                    <width>129</width>
                    <height>28</height>
                   </size>
                  </property>
                  <property name="inputMask">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPushButton" name="runButton">
                <property name="text">
                 <string>run</string>
                </property>
                <property name="iconSize">
                 <size>
                  <width>16</width>
                  <height>16</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="hogehoge" name="frame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
              <horstretch>1</horstretch>
              <verstretch>20</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2,2,1">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>158</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QTableWidget" name="tableWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>1</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
            <property name="autoScrollMargin">
             <number>32</number>
            </property>
            <property name="rowCount">
             <number>32</number>
            </property>
            <property name="columnCount">
             <number>1</number>
            </property>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>120</number>
            </attribute>
            <attribute name="verticalHeaderDefaultSectionSize">
             <number>20</number>
            </attribute>
            <attribute name="verticalHeaderMinimumSectionSize">
             <number>18</number>
            </attribute>
            <row>
             <property name="text">
              <string>r0</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r1</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r2</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r3</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r4</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r5</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r6</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r7</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r8</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r9</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r10</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r11</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r12</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r13</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r14</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r15</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r16</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r17</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r18</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r19</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r20</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r21</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r22</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r23</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r24</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r25</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r26</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r27</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r28</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r29</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r30</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r31</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>contents</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="tableWidget_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>1</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
            <property name="autoScrollMargin">
             <number>32</number>
            </property>
            <property name="rowCount">
             <number>32</number>
            </property>
            <property name="columnCount">
             <number>1</number>
            </property>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>120</number>
            </attribute>
            <attribute name="verticalHeaderDefaultSectionSize">
             <number>20</number>
            </attribute>
            <attribute name="verticalHeaderMinimumSectionSize">
             <number>18</number>
            </attribute>
            <row>
             <property name="text">
              <string>f0</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>f1</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>f2</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>f3</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>f4</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>f5</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>f6</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>f7</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>f8</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>f9</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r10</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r11</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r12</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r13</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r14</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r15</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r16</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r17</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r18</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r19</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r20</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r21</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r22</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r23</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r24</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r25</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r26</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r27</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r28</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r29</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r30</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>r31</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>contents</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>208</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>message</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="messageBuff">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1242</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>hogehoge</class>
   <extends>QFrame</extends>
   <header>hogehoge.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>runButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>runSimulation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadProgram()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>loadButtonPushed()</signal>
  <slot>runSimulation()</slot>
  <slot>stepExe(bool)</slot>
  <slot>loadProgram()</slot>
  <slot>specifySteps()</slot>
  <slot>changeStep()</slot>
 </slots>
</ui>
